engine/api/mon_db.go

```diff
@@ -103,13 +103,7 @@ func (api *API) getMonDBTimesDBQueueWorkflow(ctx context.Context, r *http.Reques
 	}
 	s1 := time.Now()
 	permissions := permission.PermissionReadExecute
-	if !isHatcheryOrWorker(r) {
-		permissions = permission.PermissionRead
-	} else {
-		usr = nil
-	}
-
-	if _, err := workflow.LoadNodeJobRunQueue(ctx, api.mustDB(), api.Cache, permissions, groupsID, usr, nil, nil, nil); err != nil {
+	if _, err := workflow.LoadNodeJobRunQueue(ctx, api.mustDB(), api.Cache, "", nil, permissions, groupsID, usr, nil, nil, nil); err != nil {
```

Comment:
too mush optional parameters
we need more specialized function of an option struct